jabref.install4j

```diff
@@ -37,155 +36,157 @@
       <envVar name="JDK_HOME" />
     </searchSequence>
     <variables>
-      <variable name="versionFourDots" value="3.1.0.0" description="" category="" />
-      <variable name="buildFileName" value="JabRef-3.1dev.jar" description="" category="" />
-      <variable name="version" value="3.1dev" description="" category="" />
+      <variable name="versionFourDots" value="3.1.0.0" />
+      <variable name="buildFileName" value="JabRef-3.1dev.jar" />
+      <variable name="version" value="3.1dev" />
     </variables>
-    <mergedProjects />
-    <codeSigning macEnabled="false" macPkcs12File="" windowsEnabled="false" windowsKeySource="pkcs12" windowsPvkFile="" windowsSpcFile="" windowsPkcs12File="" windowsPkcs11Library="" windowsPkcs11Slot="">
-      <windowsKeystoreIdentifier issuer="" serial="" subject="" />
-      <windowsPkcs11Identifier issuer="" serial="" subject="" />
-    </codeSigning>
+    <jreBundles jdkProviderId="Liberica" release="OpenJDK 11.0.4">
```

Comment:
Why is it necessary to specify that here, instead of reusing the `jlink` gradle task?